data/tooltips.txt

```diff
@@ -75,10 +75,10 @@ tip "cargo scan:"
 	`Lets you scan your target ship's cargo from up to this distance away.`
 
 tip "cargo scan power:"
-	`The range of this cargo scanner is proportional to the square root of this value.`
+	`The range of this cargo scanner is proportional to the square root of this value. The speed of the scanner also decreases in proportion to the square root of this value.`
```

Comment:
Why does power affect speed? Does this mean that if we want two scanners to have different powers but the same actual speed, we have to do a weird balancing act where we adjust the speed faster as the power goes up to maintain the same actual speed?

For that matter, is this decrease corresponding to a decrease in the square root? Or to an increase in the square root?

Reply:
An artifact of an earlier form of the PR, most likely. Speed is uncoupled from range in this build, but speed decreases with distance (and so naturally if you expect higher ranges you'll spend more time scanning)

I'll fix the tips to the best of my ability, but I'm actually not very good at writing them. I think in a specific way.

Reply:
As an example of artifacts from previous builds- scanner strength no longer depends on relative velocity to your target. I really should fix these tips.